pokemons/helpers.py

```diff
@@ -29,3 +41,32 @@ def pokemon_exists(name):
     if requests.get(url).status_code == 404:
         return False
     return True
+
+
+def get_pokemons_from_trainerteam(battle, trainer):
+    trainerteam = TrainerTeam.objects.get(
+        battle_related=battle,
+        trainer=trainer
+    )
+    return {
+        Pokemon.objects.get(name=trainerteam.pokemon_1),
+        Pokemon.objects.get(name=trainerteam.pokemon_2),
+        Pokemon.objects.get(name=trainerteam.pokemon_3),
+    }
+
+
+# task
+def save_pokeapi_pokemons():
+    url = requests.get(f'{POKEAPI_ROOT}/?limit=802')
```

Comment:
I'm not sure if there's a requirement to use all god-knows-how-many pokemons that there are, but I'd suggest reducing this number to 150 at most. It took me quite a while to populate my database, as it required over 800 requests to do so. This is also a bad practice because APIs usually have a request rate limit and this way we'd for sure be exceeding it (or very near).

Reply:
@lucabezerra I don't know many things about pokemon, so I didn't see a pattern for its uploading (considering crescent ID), because maybe I could have used that to limit the population.
I also didn't want to frustrate a user (someone who is a pokemon know-it-all, unlike me, haha) when couldn't find a match, and as the task will only run once a week in deployment environment I thought it wouldn't be a big problem.

PS. 1: Here there is a [pokemons list](https://bulbapedia.bulbagarden.net/wiki/List_of_Pok%C3%A9mon_by_base_stats_(Generation_VII-present)).
PS. 2: I saw that the girls used these 802 limit as well, because there is still more pokemons but these ones don't have a picture in API.